lib/rules/use-isnan.js

```diff
@@ -184,7 +185,35 @@ module.exports = {
                     node.arguments.length === 1 &&
                     isNaNIdentifier(node.arguments[0])
                 ) {
-                    context.report({ node, messageId: "indexOfNaN", data: { methodName } });
+
+                    /*
+                     * To retain side effects, it's essential to address `NaN` beforehand, which
+                     * is not possible with fixes like `arr.findIndex(Number.isNaN)`.
+                     */
+                    const isFixable = node.arguments[0].type !== "SequenceExpression";
```

Comment:
I think `isFixable` actually means `isSuggestable`? (To disambiguate between fixes and suggestions, it's probably better to keep the naming in line.)

Reply:
ugh... I always mix them up...
fixed (pun intended 😄)